handbook/engineering/security/goals.md

```diff
@@ -2,38 +2,51 @@
 
 ## Goals
 
-These goals represent our targeted work for 70% of our time. The remaining 30% is reserved for items that arise such as security reports. Our current work is documented in our [tracking issue](https://github.com/sourcegraph/sourcegraph/issues?q=is%3Aissue+label%3Atracking+label%3Ateam%2Fsecurity+is%3Aopen).
+These goals represent our targeted work for 70% of our time. The remaining 30% is reserved for items that arise such as security reports. Our current work is documented in a [private GitHub project](https://github.com/orgs/sourcegraph/projects/130).
```

Comment:
These goals represent 70% of our focus time. The remaining 30% is reserved for items that arise such as security reports. Our current work is documented in a [private GitHub project](https://github.com/orgs/sourcegraph/projects/130).

Reply:
Reworded, ty